### DISCUSSION TITLE
empty space above the tabs

### DISCUSSION THREAD

**Discussion by emoxam** [Eugeny/tabby#5423] [General]
Hi!
There is an empty space above the tabs, so clicking there don't change the tabs. Can we get rid of it ? So we don;t need to move mouse cursor down.
Thanks

**Comment by Eugeny** [comment]
Please include a screenshot

**Comment by emoxam** [comment]
![image](https://user-images.githubusercontent.com/3593601/148685074-79bb7a3f-8514-4740-8910-b7e917c08863.png)
Don't know if this helps.
Cursor above the tabs is regular "arrow", not the "palm with a finger", so it means cursor at the top of the screen is **not** under the tab. But above it.

**Comment by emoxam** [comment]
The same with a buttons - settings. minimize, close etc
![image](https://user-images.githubusercontent.com/3593601/148685157-2786c40a-666a-4fa7-8bf7-f6e043be0277.png)
